examples/dashboards/kustomization.yaml

```diff
@@ -20,6 +20,9 @@ configMapGenerator:
 - name: grafana-controller-resources-metrics
   files:
   - ./controller-resources-metrics.json    
+- name: dns-operator
```

Comment:
I think this needs to be `grafana-dns-operator` since that's what you have called it here https://github.com/Kuadrant/kuadrant-operator/pull/1119/files#diff-ab8359b25bb9e5ee0554a1f8eafd7e20f244e05e52837074eddbb3e0604b411b

Reply:
oh nice one, thanks!